apistructs/header.go

```diff
@@ -14,6 +14,16 @@
 
 package apistructs
 
+var (
+	EventActionCreate = "create"
```

Comment:
Bad location at `header.go`. You should put nearby `extension`.

Reply:
And can you use proto enum?

Reply:
moved to apistructs/extension.go

Reply:
> And can you use proto enum?

enum field is int32 kind, but request event by bundle is string type